corpdir-srv-sync/src/main/java/com/github/bordertech/corpdir/sync/api/mapper/ContactMapper.java

```diff
@@ -0,0 +1,41 @@
+package com.github.bordertech.corpdir.sync.api.mapper;
+
+import com.github.bordertech.corpdir.api.v1.model.Channel;
+import com.github.bordertech.corpdir.api.v1.model.Contact;
+
+/**
+ * Contact mapper
+ * @author aswinkandula
+ */
+public class ContactMapper extends AbstractVersionMapper<Contact> {
+
+	@Override
+	public void map(Contact from, Contact to) {
+		super.map(from, to);
+		to.setFirstName(from.getFirstName());
+		to.setLastName(to.getLastName());
+
+		from.getChannels().forEach((Channel fromC) -> {
+			
+			Channel toC = to.getChannels().stream()
+				.filter((Channel channel) -> channel.getValue().equals(fromC.getValue()))
+				.findAny()
+				.orElse(null);
+  
+			if (toC == null) {
```

Comment:
You might need to check this `null` test as it will be a `NullPointerException`.